components/aws/sagemaker/unit_tests/tests/test_batch_transform.py

```diff
@@ -0,0 +1,39 @@
+import json
+import unittest
+
+from unittest.mock import patch, Mock, MagicMock
+from botocore.exceptions import ClientError
+from datetime import datetime
+
+from batch_transform.src import batch_transform
+from common import _utils
+from . import testUtils
+
+
+# TODO : Errors out if model_name doesn't contain '-'
```

Comment:
Could you please create an issue in our internal tracker for this problem?

Reply:
Might also be worth it to tag this as an issue in the KFP issues? Thoughts @Jeffwan ?

Reply:
we have the internal ticket already SIM: P35195598